doc/source/whatsnew/v0.21.0.txt

```diff
@@ -385,3 +385,4 @@ Other
 - Bug in :func:`eval` where the ``inplace`` parameter was being incorrectly handled (:issue:`16732`)
 - Bug in ``.isin()`` in which checking membership in empty ``Series`` objects raised an error (:issue:`16991`)
 - Bug in :func:`unique` where checking a tuple of strings raised a ``TypeError`` (:issue:`17108`)
+- Bug in :func:`to_datetime` that cannot use tz-aware origin kwarg (:issue:`16842`)
```

Comment:
thanks, @jorisvandenbossche  I understand. I just wrote title of issue #16842. Then, should I edit like "Bug in :func:`to_datetime` that cannot use tz-aware origin kwarg raised a ``ValueError``"?